lib/request.js

```diff
@@ -242,7 +242,7 @@ module.exports = {
    */
 
   get hostname() {
-    const host = this.host;
+    const { host } = this;
```

Comment:
@maticrivo wrapping the `this.host || ''` in parens definitely reduces readability.

I'd be in favour of the de-structuring approach in both cases on this PR. Readability is maintained and consistency can be kept if another method required multiple property reads.